.trunk/.gitignore

```diff
@@ -0,0 +1,8 @@
+*out
```

Comment:
Not sure what `.trunk` is but you can add it to the .gitignore file in the Kolibri project root.

Once you do that you'll need to run `git rm -r --cached .trunk` to remove it from the git index. It _will_ keep the files on your local machine, however. [Relevant StackOverflow](https://stackoverflow.com/a/1274447)